File: src/main/java/com/todoist/pojo/UserSettings.java
package com.todoist.pojo;

public class UserSettings {
    private final boolean reminderPush;
    private final boolean reminderDesktop;
    private final boolean reminderEmail;

    public UserSettings(boolean reminderPush, boolean reminderDesktop, boolean reminderEmail) {
        this.reminderPush = reminderPush;
        this.reminderDesktop = reminderDesktop;
        this.reminderEmail = reminderEmail;
    }

    public boolean getReminderPush() {
        return reminderPush;
    }

    public boolean getReminderDesktop() {
        return reminderDesktop;
    }

    public boolean getReminderEmail() {
        return reminderEmail;
    }
}

File: src/main/java/com/todoist/pojo/UserSettings.kt
package com.todoist.pojo

open class UserSettings<
    Q : QuickAddCustomization<out Feature>,
    N : NavigationCustomization<out Feature>,
    P : HabitPushNotifications<out NotificationFeature>,
    >(
    open val reminderPush: Boolean,
    open val reminderDesktop: Boolean,
    open val reminderEmail: Boolean,
    open val completedSoundDesktop: Boolean,
    open val completedSoundMobile: Boolean,
    open val quickAddCustomization: Q?,
    open val navigationCustomization: N?,
    open val habitPushNotifications: P?,
)
